app/controllers/application_controller.rb

```diff
@@ -12,8 +12,12 @@ def new_session_path(scope)
   end
 
   def not_found!
-   render "errors/not_found", layout: "errors", status: 404
- end
+    render "errors/not_found", layout: "errors", status: 404
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.